src/scss/_mixins.scss

```diff
@@ -193,6 +193,11 @@
 			@if index($_o-grid-layout-names, $layout-name) >= index($_o-grid-layout-names, $o-grid-start-snappy-mode-at) {
 				@include oGridRespondTo($layout-name) {
 					@if $grid-mode == 'snappy' {
+						@warn 'The "snappy" grid mode is deprecated and will be ' +
```

Comment:
@notlee should this warn be removed?

Reply:
why?

Reply:
if users are using it, they should be warned not to still! :)

Reply:
WAIT WHY IS THERE TWO!? _yes_